test/index.coffee

```diff
@@ -0,0 +1,89 @@
+Promise = require 'bluebird'
+{ expect } = require 'chai'
+_ = require 'lodash'
+
+gc = require('../lib/index')
+dockerUtils = require('../lib/docker.coffee')
+
+SKIP_GC_TEST = process.env.SKIP_GC_TEST == '1' || false
+
+promiseToBool = (p) ->
+	p.return(true).catchReturn(false)
+
+pullAsync = (docker, tag) ->
+	new Promise (resolve, reject) ->
+		docker.pull tag, (err, stream) ->
```

Comment:
That's what I did at first (the `_.noop`), and it seemed like the stream wasn't draining. I'll do more testing.